qemu/tests/netkvm_rss_test.py

```diff
@@ -0,0 +1,48 @@
+from virttest import data_dir
+
+
+def run(test, params, env):
+    """
+    Test net adapter after set NetAdapterrss, this case will:
+
+    1) boot up guest
+    2) set NetAdapterrss
+    3) use speedtest trigger some network traffic
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+    def execute_command(command, timeout=60, omit=False):
+        test.log.info("command is %s" % command)
+        status, output = session.cmd_status_output(command, timeout)
+        if status != 0 and omit:
+            test.fail("execute command fail: %s" % output)
```

Comment:
It seems here `error` is properer than `fail`, as they are not really the checkpoints but setups.